### PR TITLE
Adding timer metrics for consul config watch

### PR DESCRIPTION
**Pull request by JagmohanSharma**
Integrating timer metrics for consul config watch using Netflix Servo library.

https://github.com/spring-cloud/spring-cloud-consul/issues/367

### CODE REVIEW

**Comment by spencergibb** [comment]
We shouldn't be using servo, but the new micrometer metrics

**Comment by JagmohanSharma** [comment]
I think we have java version differences in both the library. As micrometer is using java 1.8. Are we looking to make spring cloud consul to java 1.8 default?

**Comment by spencergibb** [comment]
Master is Java 8

File: pom.xml [hunk 1]
```diff
@@ -146,6 +147,12 @@
 				<artifactId>joda-time</artifactId>
 				<version>${joda-time.version}</version>
 			</dependency>
+			<!-- Micrometer dependency -->
+			<dependency>
+				<groupId>io.micrometer</groupId>
+				<artifactId>micrometer-spring-legacy</artifactId>
```
Comment (spencergibb) [inline pom.xml]: I'm a bit confused why you are pulling in the legacy support for spring boot 1.5.x. Spring Boot 2 already managed micrometer 1.0.x

Comment (JagmohanSharma) [reply pom.xml]: Updated pull request by removing legacy support dependency.

**Pull request merged**